source/virtualBuffers/gecko_ia2.py

```diff
@@ -509,6 +509,18 @@ def _getTableCellAt(self,tableID,startPos,destRow,destCol):
 		except (COMError, RuntimeError):
 			raise LookupError
 
+	def _getTableDimensions(self,tableID):
+		docHandle = self.rootDocHandle
+		table = self.getNVDAObjectFromIdentifier(docHandle, tableID)
+		try:
+			return (
+				table.IAccessibleTable2Object.nRows,
+				table.IAccessibleTable2Object.nColumns,
+			)
+		except OSError:#(COMError, RuntimeError, AttributeError):

```

Comment:
I don't see how `OSError` may be raised by this code. Could you elaborate? At a guess this should be a `COMError`.

Reply:
Good catch! Deleted OSError.